src/main/java/pl/allegro/tech/discovery/consul/recipes/watch/catalog/Services.java

```diff
@@ -12,11 +13,11 @@ public Services(Map<String, List<String>> serviceNamesToTags) {
         this.serviceNamesToTags = serviceNamesToTags;
     }
 
-    boolean containsService(String serviceName) {
+    public boolean containsService(String serviceName) {
```

Comment:
po co ta metoda ma byc publiczna?

Reply:
> po co ta metoda ma byc publiczna? (why is this method supposed to be public?)
Services is simple wrapper around Map, I don't see any point in hiding such methods which might be useful in some context.